src/Calculator/Views/TitleBar.xaml

```diff
@@ -32,25 +37,22 @@
                    Foreground="{ThemeResource TitleBarForegroundBaseHighBrush}"
                    FontSize="12"
                    TextAlignment="Left"
-                   TextTrimming="CharacterEllipsis"
-                   Visibility="{x:Bind ApplicationViewModel.IsAlwaysOnTop, Converter={StaticResource BooleanToVisibilityNegationConverter}, Mode=OneWay}"/>
+                   TextTrimming="CharacterEllipsis"/>
         </Grid>
-        <Button x:Name="AoTAlwaysOnTopButton"
+        <Button x:Name="ExitAlwaysOnTopButton"
                 x:Uid="ExitAlwaysOnTopButton"
-                Grid.Row="0"
+                x:Load="False"
                 FontFamily="{StaticResource CalculatorFontFamily}"
                 Style="{ThemeResource CommandBarFlyoutEllipsisButtonStyle}"
                 Content="&#xEE47;"
-                Click="AlwaysOnTopButtonClick"
+                Click="AlwaysOnTopButton_Click"
                 Width="46"
-                Height="32"
+                Height="Auto"
+                Visibility="Collapsed"
                 HorizontalAlignment="Left"
                 HorizontalContentAlignment="Center"
                 Background="Transparent"
-                FontWeight="Thin"
```

Comment:
The button looks bold now, I think you still want the FontWeight so that the style of the two AOT buttons match.

Reply:
My fault! Fixed!